skxray/fitting/models.py

```diff
@@ -53,20 +53,19 @@
 from lmfit.models import GaussianModel as LmGaussianModel
 from lmfit.models import LorentzianModel as LmLorentzianModel
 from .lineshapes import (elastic, compton, gaussian_tail, gausssian_step,
-                         lorentzian2, gaussian, lorentzian
-)
-
-import logging
-logger = logging.getLogger(__name__)
+                         lorentzian2, gaussian, lorentzian)
 
 from skxray.fitting.lineshapes import (elastic, compton, gaussian,
                                        lorentzian, lorentzian2)
-
 from skxray.fitting.base.parameter_data import get_para
 
+import logging
+logger = logging.getLogger(__name__)
+
 
 def set_default(model_name, func_name):
```

Comment:
Because sometimes I am not sure about the bound_type, which is given by user or from configuration file. For different bound_types, such as 'none' or 'fixed', the arguments passing to Model.set_param_hint vary. I just write a simple factory to handle this, otherwise this patter will repeat many times.